apps/judicial-system/web/src/utils/hooks/useCaseList/index.tsx

```diff
@@ -141,8 +141,12 @@ const useCaseList = () => {
   )
 
   const handleOpenCase = useCallback(
-    (id: string, openInNewTab?: boolean) => {
-      Promise.all(timeouts.map((timeout) => clearTimeout(timeout)))
+    async (id: string, openInNewTab?: boolean) => {
+      const clearTimeouts = () => {
+        timeouts.map((timeout) => clearTimeout(timeout))
```

Comment:
No need to create a consolidated promise as we are not waiting for its completion.